var/spack/repos/builtin/packages/mesa/package.py

```diff
@@ -106,7 +151,29 @@ def configure_args(self):
                 args.append('--disable-llvm-shared-libs')
             args.append('--with-llvm-prefix=%s' % spec['llvm'].prefix)
 
-        if drivers:
-            args.append('--with-gallium-drivers=' + ','.join(drivers))
+        args.append('--with-gallium-drivers=' + ','.join(drivers))
```

Comment:
It appears that ```drivers``` can be empty if ```~swrender``` and ```~hwrender``` are set, does that not matter though? i.e. maybe it's fine to specify ```--with-gallium-drivers=```

Reply:
It is OK to specify `--with-gallium-drivers=`, which is handled the same way as `--without-gallium-drivers`. But it would be clearer to specify `--without-gallium-drivers` if no drivers are requested, so I will fix that.